client/state/plugins/installed/selectors-ts.ts

```diff
@@ -321,3 +322,89 @@ export const getSiteObjectsWithoutPlugin = createSelector(
 	( state: AppState ) => [ getAllPluginsIndexedByPluginSlug( state ), getSitesItems( state ) ],
 	( state: AppState, siteIds: number[], pluginSlug: string ) => [ pluginSlug, ...siteIds ].join()
 );
+
+export function getStatusForPlugin( state: AppState, siteId: number, pluginId: string ) {
+	if ( typeof state.plugins.installed.status[ siteId ] === 'undefined' ) {
+		return false;
```

Comment:
A boolean is an unexpected value from this selector IMHO. Should it be `null` or `undefined` instead?

Reply:
I made it return `undefined`.